common/challenge.mk

```diff
@@ -0,0 +1,23 @@
+INTERNAL_PORT := 80
```

Comment:
From my point of view this should be `?=`. Same for the others.

Reply:
Yes. `:=` is wrong because it overrides the variabes set in the including Makefile.

Reply:
Done.